### PR TITLE
Ask CFPB: Send Virtual Pageview events on answer pages

### PR DESCRIPTION
**Pull request by virginiacc**
Send Analytics Virtual Pageview events on Ask answer pages

## Additions

- Send analytics event with category, title, and action (configured to match old url structure) on answer pages  

## Testing

1. Check out branch and run `gulp scripts`
2. Navigate to [English](http://localhost:8000/ask-cfpb/what-information-do-i-have-to-provide-a-lender-in-order-to-receive-a-loan-estimate-en-1987/) and [Spanish](http://localhost:8000/es/obtener-respuestas/me-gustaria-poder-tener-la-ayuda-de-mis-amigos-o-familiares-con-el-pago-de-mis-facturas-y-actividades-bancarias-cuales-son-mis-opciones-es-1145/) answer pages and inspect the `dataLayer` object in console to verify that pageview event has been added

## Screenshots
screen shot 2017-06-15 at 11 22 24 am
screen shot 2017-06-15 at 11 19 08 am

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

File: cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html [hunk 1]
```diff
@@ -93,3 +93,11 @@ <h4 class="supplement-title">Preguntas similares</h4>
         </span>
     </div><!-- .share -->
 {% endblock %}
+
+{% block page_js %}
+<script type="text/javascript">
+    var answerID = '{{ answer_id }}';
```
Comment (sebworks) [inline cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html]: ok, no globals please.

Comment (virginiacc) [inline cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html]: I can pass it as a data attribute

Comment (sebworks) [inline cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html]: yeah, that sounds good.

File: cfgov/unprocessed/js/modules/Analytics.js [hunk 2]
```diff
@@ -44,7 +48,10 @@ var Analytics = {
         },
         set: function( value ) {
           _tagManager = value;
-          Analytics.tagManagerIsLoaded = true;
+          if ( !Analytics.tagManagerIsLoaded ) {
```
Comment (sebworks) [inline cfgov/unprocessed/js/modules/Analytics.js]: nice

File: cfgov/unprocessed/js/modules/Analytics.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
 'use strict';
 
 var isArray = require( './util/type-checkers' ).isArray;
-
+var EventObserver = require( '../modules/util/EventObserver' );
+var eventObserver = new EventObserver();
 var Analytics = {
 
   tagManagerIsLoaded: false,
+  addEventListener: eventObserver.addEventListener,
```
Comment (sebworks) [inline cfgov/unprocessed/js/modules/Analytics.js]: Could you use `Object.Assign( Analytics, eventObserver )`?

File: cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html (hunk 1 above)
Comment (virginiacc) [inline cfgov/jinja2/v1/ask-cfpb/answer-page-spanish.html]: Updated in https://github.com/cfpb/cfgov-refresh/pull/3039/commits/b87358e2217322f64d9ffdaacc166f0803502a16. I'm using an empty div to pass the data for now, but I'll think about whether there's a better place for it.

**Comment by higs4281** [comment]
👍 shows up for me!

**Pull request merged**